std/format/package.d

```diff
@@ -198,66 +198,81 @@ contain exactly one $(I format specifier), which specifies the way,
+
+             In case of $(B 'r') it denotes to use big endian.))
+   $(TR $(TD $(B '0'))
+        $(TD Applies only to numbers that are printed right justified.
+             If the zero flag is present, the space left to the number is
+             filled with zeros instead of spaces.))
+   $(TR $(TD $(B '#'))
+        $(TD An alternativ output shall be used. This depends on the type
+             to be formatted and the $(I format character) used. See the
+             sections below for more information.))
+)
 
-    $(BOOKTABLE Flags affect formatting depending on the specifier as
-    follows., $(TR $(TH Flag) $(TH Types&nbsp;affected) $(TH Semantics))
+$(SECTION4 Width\, Precision and Separator)
 
-    $(TR $(TD $(B '-')) $(TD numeric, bool, null, char, string, enum, pointer) $(TD Left justify the result in
-        the field.  It overrides any $(B 0) flag.))
+The $(I width) specifies the minimum width of the result.
 
-    $(TR $(TD $(B '+')) $(TD numeric) $(TD Prefix positive numbers in
-    a signed conversion with a $(B +).  It overrides any $(I space)
-    flag.))
+The meaning of $(I precision) depends on the format indicator. For
+integers it denotes the minimum number of digits printed, for
+real numbers it denotes the number of fractional digits and for
+strings and compound types it denotes the maximum number of elements,
+that are included in the output.
 
-    $(TR $(TD $(B '#')) $(TD integral ($(B 'o'))) $(TD Add to
-    precision as necessary so that the first digit of the octal
-    formatting is a '0', even if both the argument and the $(I
-    Precision) are zero.))
+A $(I separator) is only used for formatting numbers. If it is
+specified, the output is divided into chunks of three digits,
+separated by a $(B ','), for better readability. The number of digits
+in a chunk can also be given explicitly.
 
-    $(TR $(TD $(B '#')) $(TD integral ($(B 'x'), $(B 'X'))) $(TD If
-       non-zero, prefix result with $(B 0x) ($(B 0X)).))
+In all three cases the number can be replaced by a $(B '*'). In that
+case the next argument is used as the number. If the argument is a
+negative number, in case of $(I width), the $(B '-') flag is applied,
+in case of $(I precision) and $(I separator) it is, as if no
+precision or separator is specified.
 
-    $(TR $(TD $(B '#')) $(TD floating) $(TD Always insert the decimal
-       point and print trailing zeros.))
+The $(I separator) can also be followed by a $(B '?'). In that case
+an additional argument is used to specify the symbol that should be
+used to separate the chunks.
 
-    $(TR $(TD $(B '0')) $(TD numeric) $(TD Use leading
-    zeros to pad rather than spaces (except for the floating point
-    values `nan` and `infinity`).  Ignore if there's a $(I
-    Precision).))
+$(SECTION4 Position)
 
-    $(TR $(TD $(B ' ')) $(TD numeric) $(TD Prefix positive
-    numbers in a signed conversion with a space.)))
+Normally, the arguments are processed in order. With the $(I
```

Comment:
```suggestion
By default, the arguments are processed in order. With the $(I
```

Reply:
What kind of order do you refer to? Lexical order?

Reply:
From left to right (the order given in the function call).